src/Tizen.NUI/src/public/Layouting/GridLayout.cs

```diff
@@ -22,7 +22,7 @@
 namespace Tizen.NUI
 {
     /// <summary>
-    /// [Draft] This class implements a grid layout
+    /// This class implements a grid layout
```

Comment:
Please write something more, because its purpose is already suggested in the class name.

Reply:
DONE